README.md

```diff
@@ -29,39 +28,35 @@ A distinctive feature of AssertiveTS with other assertion libraries is that it l
 - Works with any test runner and framework such as [Jest](/docs/jest-tutorial.md), [Mocha](/docs/mocha-tutorial.md), or Ava
 - Well tested: more than 300 tests!
 
-## Install
-
-```sh
-npm install --save-dev @stackbuilders/assertive-ts
-```
+## Packages
 
-Or:
+For conviniece, this library is split into package grouped within the same namespace:
 
-```sh
-yarn add --dev @stackbuilders/assertive-ts
-```
+- **[assertive-ts/core](https://github.com/stackbuilders/assertive-ts/blob/main/packages/core/README.md):** Core functionalities, assertions applicable for any kind of application. This package is required to use the [extension mechanism](https://github.com/stackbuilders/assertive-ts/blob/main/packages/core/README.md#extension-mechanism-⚙️) (plugins). This is package is the replacement for the deprecated `stackbuilders/assertive-ts` package.
 
 ## Usage
 
-Import the library in your test script:
+Using you favorite test runner, you just need to import the `expect` and test away! If you don't really agree with `expect` as the name of the assertion function, we provide a couple aliases to the function, such as `assert` and `assertThat`.
```

Comment:
```suggestion
Using you favorite test runner, you just need to import the `expect` and test away! If you don't really agree with `expect` as the name of the assertion function, we provide a couple aliases such as `assert` and `assertThat`.
```

Reply:
This suggestion here may not be worth as the same paragraph is duplicated in `core/readme`, and accepting this change my imply keeping the other one in sync with this one, so, up to you.

Reply:
Makes sense, we have this part duplicate in the two readme files to provide better visibility of how the library works in GitHub as well as NPM 🙂